ext/liquid_c/extconf.rb

```diff
@@ -1,5 +1,5 @@
 require 'mkmf'
-$CFLAGS << ' -Wall -Werror -Wextra -Wno-unused-parameter -Wno-missing-field-initializers'
+$CFLAGS << ' -std=c99 -Wall -Werror -Wextra -Wno-unused-parameter -Wno-missing-field-initializers'
```

Comment:
why was this needed?

Reply:
I started using `if (long i ...` which was added in c99.